### ISSUE TITLE
[Warning] Access denied for user '_5e5899d8398b5f7b'@'172.28.0.8' (using password: YES)

### ISSUE THREAD

**Issue by Tipitopi79** [frappe/frappe_docker#1474]
Hello, I have a problem when installing Erpnext on my Synology in the docker. I didn't change anything on the one shown below, which should actually start without any changes. I also waited about 15 minutes until I tried to access the web interface.
I get the following error in the container:
`[Warning] Access denied for user '_5e5899d8398b5f7b'@'172.28.0.8' (using password: YES)`
Here is the whole log of this container:

```
2024-09-22 11:03:51+00:00 [Note] [Entrypoint]: MariaDB upgrade not required
2024-09-22 11:03:51 0 [Note] Starting MariaDB 10.6.19-MariaDB-ubu2004 source revision 8f020508c87461bbeee551c3001acac56d506ae7 server_uid /6j7Q4z54VID6OsIdOY43YZPiAI= as process 1
2024-09-22 11:03:51 0 [Note] InnoDB: Compressed tables use zlib 1.2.11
2024-09-22 11:03:51 0 [Note] InnoDB: Number of pools: 1
2024-09-22 11:03:51 0 [Note] InnoDB: Using crc32 + pclmulqdq instructions
2024-09-22 11:03:51 0 [Note] InnoDB: Using Linux native AIO
2024-09-22 11:03:51 0 [Note] InnoDB: Initializing buffer pool, total size = 134217728, chunk size = 134217728
2024-09-22 11:03:51 0 [Note] InnoDB: Completed initialization of buffer pool
2024-09-22 11:03:52 0 [Note] InnoDB: Starting crash recovery from checkpoint LSN=23467498,23467498
2024-09-22 11:03:56 0 [Note] InnoDB: 128 rollback segments are active.
2024-09-22 11:03:56 0 [Note] InnoDB: Removed temporary tablespace data file: "./ibtmp1"
2024-09-22 11:03:56 0 [Note] InnoDB: Creating shared tablespace for temporary tables
2024-09-22 11:03:56 0 [Note] InnoDB: Setting file './ibtmp1' size to 12 MB. Physically writing the file full; Please wait ...
2024-09-22 11:03:56 0 [Note] InnoDB: File './ibtmp1' size is now 12 MB.
2024-09-22 11:03:56 0 [Note] InnoDB: 10.6.19 started; log sequence number 23467510; transaction id 7563
2024-09-22 11:03:56 0 [Note] Plugin 'FEEDBACK' is disabled.
2024-09-22 11:03:56 0 [Warning] You need to use --log-bin to make --expire-logs-days or --binlog-expire-logs-seconds work.
2024-09-22 11:03:56 0 [Note] Server socket created on IP: '0.0.0.0'.
2024-09-22 11:03:56 0 [Note] Server socket created on IP: '::'.
2024-09-22 11:03:56 0 [Note] InnoDB: Loading buffer pool(s) from /var/lib/mysql/ib_buffer_pool
2024-09-22 11:03:58 0 [Note] mariadbd: ready for connections.
Version: '10.6.19-MariaDB-ubu2004'  socket: '/run/mysqld/mysqld.sock'  port: 3306  mariadb.org binary distribution
2024-09-22 11:04:03 6 [Warning] Aborted connection 6 to db: 'unconnected' user: 'unauthenticated' host: '172.28.0.10' (This connection closed normally without authentication)
2024-09-22 11:04:14 0 [Note] InnoDB: Buffer pool(s) load completed at 240922 11:04:14
2024-09-22 11:05:12 54 [Warning] Access denied for user '_5e5899d8398b5f7b'@'172.28.0.8' (using password: YES)
2024-09-22 11:06:13 102 [Warning] Access denied for user '_5e5899d8398b5f7b'@'172.28.0.8' (using password: YES)
2024-09-22 11:07:13 154 [Warning] Access denied for user '_5e5899d8398b5f7b'@'172.28.0.8' (using password: YES)
2024-09-22 11:08:13 205 [Warning] Access denied for user '_5e5899d8398b5f7b'@'172.28.0.8' (using password: YES)
```

[https://github.com/frappe/frappe_docker/blob/main/pwd.yml](url)

All containers run except for erpnext-configurator-1 and erpnext-creat-site-1

Here are the last lines from the container erpnext-creat-site-1:
```
===========] 100%
Updating customizations for Address
Updating customizations for Contact
Updating Dashboard for erpnext
*** Scheduler is disabled ***
Current Site set to frontend
```

My web interface is then probably not available, it is "503 Service Unavailable
No server is available to handle this request."displayed

Can someone help me?

**Comment by revant** [comment]
> All containers run except for erpnext-configurator-1 and erpnext-creat-site-1

https://github.com/frappe/frappe_docker/blob/main/docs/single-compose-setup.md#run-once-configuration

Try,
https://github.com/frappe/frappe_docker/blob/main/docs/troubleshoot.md#fixing-mariadb-issues-after-rebuilding-the-container

**Comment by Tipitopi79** [comment]
I had made a stupid mistake. I usually always mount all volumes on my synology. But with this compose file I left everything as it was for testing purposes. I had deleted all containers several times and tried to restart them again and again. However, I then saw in Portainer that the volumes managed by Docker were still there. After I deleted them and reinstalled everything again, it worked. Then I mounted all the volumes on my synology. anyway, thank you for your help.
